serveur_calcul_python/tests/unit/test_conversion_unite.py

```
@@ -0,0 +1,32 @@
+#sys.path.append(str(Path(__file__).resolve().parents[2]))
+
+import classes.parking_inventory_inputs as PCI
+from serveur_calcul_python.tests.data_gen.tax_data_store import generate_tax_dataset
+from serveur_calcul_python.tests.data_gen.regulation_set_data_store import generate_parking_regulation_sets
+from serveur_calcul_python.tests.data_gen.calcs_input_data_store import generate_prs_1_inputs,generate_prs_2_inputs,generate_prs_3_inputs
+
+def test_conversion_unite():
+    tax_dataset_prs_1  = generate_tax_dataset(1)
+    tax_dataset_prs_2 = generate_tax_dataset(2)
+    tax_dataset_prs_3 = generate_tax_dataset(3)
+    prs_1,prs_2,prs_3,prs_4 = generate_parking_regulation_sets()
```

Ruff (0.15.12)

[warning] Unpacked variable `prs_4` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)